presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -47,6 +51,22 @@ public BaseJdbcConfig setConnectionUrl(String connectionUrl)
         return this;
     }
 
+    public Set<String> getJdbcTypesMappedToVarchar()
```

Comment:
Add getter/setter in same order you added the field (ie field is last, so move getter/setter to the end of the class)